scripts/index.js

```diff
@@ -5,16 +5,25 @@
 // Add values to the values array and see what happens :)
 var TotalCount = 0;
 var Sunday,Monday,Tuesday,Wednesday;
-Sunday = 3200;
+Sunday = 0;
 Monday = 0;
 Tuesday = 0;
 Wednesday = 0;
 var values = [Sunday, Monday, Tuesday, Wednesday];
-
 TotalCount = values.reduce(function(e1, e2) { return e1 + e2; });
-
 document.getElementById('totalCount').innerHTML = TotalCount;
 
+var ref = firebase.database().ref();
+ref.on("donations-f206b", function(snapshot){
+  var newDonations = snapshot.val();
+    Sunday = newDonations.sundayDonations;
+    Monday = newDonations.mondayDonations;
+    Tuesday = newDonations.tuesdayDonations;
+    Wednesday = newDonations.wednesdayDonations;
+    TotalCount = values.reduce(function(e1, e2) { return e1 + e2; });

```

Comment:
i guess you need to add `var values = [Sunday, Monday, Tuesday, Wednesday];` again before line 23